### DISCUSSION TITLE
Can't disable virtual text for diagnostics

### DISCUSSION THREAD

**Discussion by gmnz** [LazyVim/LazyVim#444] [Q&A]
Can't disable virtual text for diagnostics:

```lua
vim.diagnostic.config({
  virtual_text = false,
  -- float = {
  --   source = 'always',
    -- border = border
  -- },
})
```

in lua/config/options.lua doesn't disable it... It's still showing. What am I doing wrong?

**Comment by dpetka2001** [comment]
It's an option for lspconfig, so you should probably put it in a spec about "neovim/nvim-lspconfig" instead of `options.lua`.

**Reply by gmnz** [reply]
How exactly would that look? Looks like a builtin nvim variable to me, not an lspconfig setup option...

**Reply by folke** [reply]
Have you tried reading the docs? https://www.lazyvim.org/plugins/lsp#nvim-lspconfig

**Reply by folke** [reply]
There's also a search bar. So you could have searched for "diagnostics"

**Reply by gmnz** [reply]
Oh, sorry. Yeah, went through the docs many times, but didn't see this one or didn't expect it to be where it was. Yeah, the search bar works great, if one comes to the idea of using it. Thank you and sorry again! Next time I'll search more thoroughly. :-)
